### PR TITLE
DPOR 2072 - add contractions

### PR DESCRIPTION
**Pull request by kriher18**

### CODE REVIEW

File: 010_About_Style_And_Standards.html.md [hunk 1]
```diff
@@ -84,6 +84,15 @@ Always verify the use of the articles "a", "an", and "the" where appropriate. Us
 
 For more information, see the [Articles in English Grammar](https://github.com/YaaS/REST_API_Documentation_Guidelines/wiki//005-Articles-in-English-Grammar) blog post.
 
+##### Contractions
+
+It is okay to use contractions for a people-centric approach, but avoid using ones that may be difficult for non-native English speakers to understand. Specifically, do not use the following:
+
+:no_entry: Any will contractions, such as you’ll or we’ll.<br>
```
Comment (tszafranska) [inline 010_About_Style_And_Standards.html.md]: The update looks good! The only question I have is, do we want to be consistent and place the examples in quotes, like we did in the Pronouns section? So for example, "Any "will" contractions, such as "you’ll" or "we’ll"?

**Comment by tszafranska** [comment]
I just noticed that the Table of Contents also needs to be updated with the Contractions section.

File: 010_About_Style_And_Standards.html.md (hunk 1 above)
Comment (kriher18) [reply 010_About_Style_And_Standards.html.md]: Yes, that is a good point. I will update it.

**Comment by kriher18** [comment]
I will also add this to the TOC.

**Pull request merged**